### DISCUSSION TITLE
Option to mass clone Organization items to other Organizations or Vaults

### DISCUSSION THREAD

**Discussion by jpdsc** [dani-garcia/vaultwarden#3213] [Ideas]
Today I had to move items from an Organization to my own vault because this organization will not being used anymore.
I spend 45 minutes cloning all the items one by one to my own vault to then delete them from the Organization and delete the Organization itself.

Perhaps an idea to add an option to mass clone items in an organization to speed up this process. 
A good solution would be to add:

- Option to mass clone items
- Option to choose the text which comes after the name of the cloned item. Current behavior =  "- Clone"

Thanks!

**Comment by BlackDex** [comment]
These are client side items, which Vaultwarden doesn not maintain.

You could have made your life easier if you would have exported it instead and then imported that again somewhere else (at least i think that it is possible to export from org and import into user account).

**Reply by jpdsc** [reply]
Ah! Ok, I wasn't aware this was client side.
Oh man! I completely forgot about the exporting process. It is indeed possible!

Thank you for the suggestion and keep up the great work with VW! We love it!

**Reply by BlackDex** [reply]
Thanks for the confirmation regarding the org-export > user-import!

**Comment by tessus** [comment]
I also find it very annoying that one cannot move an item to a personal vault. I thing that was a bad design decision from Bitwarden. I am mot sure if anyone has requested this on the Bitwarden forum yet.

**Reply by jpdsc** [reply]
> I also find it very annoying that one cannot move an item to a personal vault. I thing that was a bad design decision from Bitwarden. I am mot sure if anyone has requested this on the Bitwarden forum yet.

I think this is done intentional. The Organization feature has an admin who manages the Organization. If you allow multiple people to move data out the Organization then it might become a mess.
If you have low amount of data to move is cloning the best option. With more data, export everything from Organization and then Import as user. They will automatically go in the No folder, folder and from there you can delete what isn't needed.

**Reply by tessus** [reply]
> If you allow multiple people to move data out the Organization then it might become a mess.

That I understand and I am not saying that this should be possible by a user. However, an admin should be able to move it out of there....

**Reply by jpdsc** [reply]
> > If you allow multiple people to move data out the Organization then it might become a mess.
> 
> That I understand and I am not saying that this should be possible by a user. However, an admin should be able to move it out of there....

Yes, I completely agree. Perhaps good feedback from the Bitwarden team if no one did already.

**Reply by kpfleming** [reply]
If an admin could do it, they'd only be able to move the item(s) to their own personal vault.

**Reply by tessus** [reply]
yep, but this is still ok - as the OP seems to have been the admin of that org.

**Reply by BlackDex** [reply]
The admin could also share that export via `Send` if they really need to. But i would consider that a breach in security actually.
